### ISSUE TITLE
Confocal_GUI_blank_colorbar

### ISSUE THREAD

**Issue by abhishek-kejriwal** [Ulm-IQO/qudi#705]
I am trying to obtain confocal images using the confocal GUI. Even though I am getting clear confocal images with an appropriate colormesh, the colorbar only shows the tick labels without any color. I am having trouble fixing this issue. 

Reference image:
 
<img width="722" alt="GetImage" src="https://user-images.githubusercontent.com/81922897/196440298-645be75d-9cb7-491f-a3d2-ba399d9fd8d9.png">

**Comment by kiotex** [comment]
I faced the same problem and solved it quite some time ago..
Uninstalling pyqtgraph and pyqtgraph-qo and then re-installing pyqtgraph-qo solve this problem.

The latest version of pyqtgraph is version 0.13.1 and pyqtgraph-qo seems to be a development version of version 0.10.X.
The problem occurred when I installed either version 0.11.0 or later of pyqtgraph or version 0.10.0.6 or later of pyqtgraph-qo.

It seems that some specification change in pyqtgraph's QtGui from version 0.10.0.5 to 0.10.0.6, on which ColorBar() in qudi/gui/guiutils.py, which handles the color bar on the GUI side, depends, caused this bug.
